lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -1236,6 +1236,9 @@ namespace Js
 #if DBG
         void DumpLibraryByteCode();
 #endif
+
+        Var experimentalVersion; // Wasm.experimentalVersion
```

Comment:
Could you put this next to the `wasmObject` inside the `#ifdef ENABLE_WASM` (line 380) of this file